examples/sample-pub/publisher.js

```diff
@@ -133,6 +133,12 @@ function startFlow(flow, var_args) {
 function startFlowAuto() {
   let flow = (window.location.search || '').split('?')[1] || 'demo';
 
+  // Provide a shortcut to metering params.
+  if (location.search === '?metering') {
```

Comment:
This isn't consistent with above. If `window.location` is prior convention and prevalent, probably should continue as such. Or change above to match new convention.

Reply:
Good catch, fixed !